auslib/test/test_db.py

```diff
@@ -3256,7 +3256,7 @@ def testHasPermissionNotAllowedByProduct(self):
 
     def testGetUserRoles(self):
         got = self.permissions.getUserRoles("bob")
-        self.assertEquals(set(got), set(["releng", "dev"]))
+        self.assertEquals(got, [{'data_version': 1, 'role': u'dev'}, {'data_version': 1, 'role': u'releng'}])
```

Comment:
Please continue using a set here - the order in which these are returned from the database is not guaranteed.